ng2-components/ng2-alfresco-documentlist/src/data/share-datatable-adapter.ts

```diff
@@ -153,7 +153,7 @@ export class ShareDataTableAdapter implements DataTableAdapter {
         this.sorting = sorting;
 
         if (sorting.key && sorting.key.includes('sizeInBytes')) {
```

Comment:
can we remove this since it's resolved in [sortRows](https://github.com/Alfresco/alfresco-ng2-components/pull/2594/files#diff-671b3b5553c649a3c9acf935352624ccR184) ?